generation.py

```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+def generate_matrices():
+    width = randint(1, 7)
+    height = randint(1, 7)
```

Comment:
why we limited with 7 ? extend it. May be allow to specify matrix sizes from user input